src/auto-evo/mutation_strategy/AddOrganelleAnywhere.cs

```diff
@@ -86,15 +86,17 @@ public static AddOrganelleAnywhere ThatConvertBetweenCompounds(Compound fromComp
 
         var mutated = new List<Tuple<MicrobeSpecies, float>>();
 
+        // TODO: reuse this memory somehow
         var workMemory1 = new List<Hex>();
         var workMemory2 = new List<Hex>();
+        var workMemory3 = new HashSet<Hex>();
```

Comment:
Would making them attributes of the class and just clearing them at the start work? It would keep the rules of no state due to them being reset each time the function is called.